core/src/main/java/org/axonframework/commandhandling/disruptor/DisruptorCommandBus.java

```diff
@@ -635,6 +634,19 @@ public DisruptorRepository(Class<T> type) {
         public Aggregate<T> newInstance(Callable<T> factoryMethod) throws Exception {
             return CommandHandlerInvoker.<T>getRepository(type).newInstance(factoryMethod);
         }
+
+        @Override
+        public void send(Message<?> message, ScopeDescriptor scopeDescription) throws Exception {
+            if (scopeDescription instanceof AggregateDescriptor) {
+                load(((AggregateDescriptor) scopeDescription).getIdentifier().toString()).handle(message);
```

Comment:
loading an aggregate may result in an Exception. In the case of an `AggregateNotFoundException`, we'd want to silently ignore (but log on debug level) the message. It is not uncommon for an Aggregate to be deleted before a deadline expires.

Reply:
With message, do you mean the `message` parameter to be send, or the `message` of the exception?